android/build.gradle

```diff
@@ -44,6 +45,23 @@ repositories {
 }
 
 dependencies {
+    //noinspection GradleDynamicVersion
```

Comment:
nit: let's have this comment be consistent with the one on L42 - going to start putting together a style guide on comments / release notes

Reply:
@jsani-radar this "noinspection" comment is a keyword used by the linter, and is not a standard comment. There are a few things like this - noinspection, region/end region, etc, that we can call out in the style guide.